expression/builtin_time_vec_test.go

```diff
@@ -119,7 +119,9 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 			geners:        []dataGenerator{&unitStrGener{}, nil, nil},
 		},
 	},
-	ast.TimestampDiff:    {},
+	ast.TimestampDiff:    {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString, types.ETDatetime, types.ETDatetime}},
```

Comment:
Please consider using [`constStrGener`](https://github.com/pingcap/tidb/blob/498a7f6d009ca8c8999f922b9b0f2bcae6fdded3/expression/bench_test.go#L616) to generate the first argument

Reply:
> Please consider using [`constStrGener`](https://github.com/pingcap/tidb/blob/498a7f6d009ca8c8999f922b9b0f2bcae6fdded3/expression/bench_test.go#L616) to generate the first argument



It seems that unitStrGener would be more proper for this test as it is used to generate strings which are unit format.